### DISCUSSION TITLE
Configuration support

### DISCUSSION THREAD

**Discussion by LouDnl** [LouDnl/USBSID-Pico#4] [Q&A]
Any questions regarding configuration of USBSID can be asked here

**Comment by micv2** [comment]
Hi LouD,

is in the config mirrored play = Socket 2 Act as One? Why am I asking? I have 2x 6581 on my board. If I activate Pseudo Stereo in the configuration, Warhawk.sid is nice and powerful on both channels. On the other hand, I can't hear anything with Commando.sid, why is that?

Could pseudo stereo be automated? If a tune for one SID is running, it automatically switches to pseudo stereo. That way the sound would come from both channels instead of just one. With dual-SID tune, both SIDs would automatically activate for both channels. Would such an auto-detect function be possible?

Best regards,
MiC

**Reply by LouDnl** [reply]
Hi Mic,

Thanks for your question. PCB version 1.0 has no option to switch back and forth between stereo and mono. It has a jumper to either use sid 1 for left and right or use sid 1 for right and sid 2 for left channels.

What I personally use is a 3-way jumper cap to connecf all 3 audio jumper pins together. This creates a mono signal by combining the audio at the output pins. Adding a jumper wire between pin S1 and S2 near the audio jack would do the same.
Needless to say, this comes at some audio loss and potential danger of breaking sids, do this at your own risk! (No issues here yet).

Another option would be when using 2 sids is to have the pseudo stereo created by the player that plays the tune by changing usbsid firmware settings on the fly. The firmware supports it and has commands ready for this, except I have not implemented this anywhere (yet). Feel free to create thia yourself ofcourse. 

PCB version 1.2 (testing board) has a digital switch to change mono to stereo on the fly by changing the outputs.
This will be released on PCB 1.3 which I am currently designing.
There is no player other then the config website that supports this atm.

**Reply by LouDnl** [reply]
> Hi LouD,
> 
> is in the config mirrored play = Socket 2 Act as One? Why am I asking? I have 2x 6581 on my board. If I activate Pseudo Stereo in the configuration, Warhawk.sid is nice and powerful on both channels. On the other hand, I can't hear anything with Commando.sid, why is that?
> 
> Could pseudo stereo be automated? If a tune for one SID is running, it automatically switches to pseudo stereo. That way the sound would come from both channels instead of just one. With dual-SID tune, both SIDs would automatically activate for both channels. Would such an auto-detect function be possible?
> 
> Best regards,
> MiC

Regarding some tunes not producing audio, is this on the config website using webusb?

**Comment by micv2** [comment]
Thank you LouD. I did it as you suggested; it works really well. A great project, I really like it 👍 

Best regards, 
MiC

**Reply by LouDnl** [reply]
Good to hear! Glad you like it 😁

**Comment by matt199394** [comment]
Hi LouD,
I have copied the udev permission in /etc/udev/rules.d and  reload the udev rules as explained in the software manual. 
When I try to connect USBSIDPico using the test page I receive the message "Autoconnect not actived yet, please connect to a WebUSB device first."
Any suggestions?
My OS is Ubuntu Ubuntu 24.04

Thanks and best regards

Matteo

**Reply by LouDnl** [reply]
What browser are you using? The configtool @ usbsid.loudai.nl only works with Chrome based browsers.
Otherwise use the commandine tool.

**Comment by micv2** [comment]
Hi, 
it doesn't work in Firefox. Try Chrome.

Regards,
MiC

**Comment by matt199394** [comment]
Hi,
It works fine with Chrome :), thanks.
It works with jsidplay2, but the tunes are played too fast.
The RGB LED remains off. I checked the configuration and firmware version, and everything seems fine.

**Reply by LouDnl** [reply]
Better use Vice for now, jsidplay2 is still in testing

**Reply by LouDnl** [reply]
Can u post a photo of the pico pcb nnear the rgb led and a screenshot of your config

**Comment by matt199394** [comment]
![20250728_092005](https://github.com/user-attachments/assets/825fe036-6b29-4cdc-b3c8-057764ff04e2)
![20250728_091401](https://github.com/user-attachments/assets/92d5a625-743f-4d40-a6af-99c74852815d)

**Reply by LouDnl** [reply]
Thanks, if you look at the pico pcb at thr R68 solder jumper near the RGB led, you can see that it's open. If you want the RGB led to work, close it with some solder :)

**Reply by matt199394** [reply]
Thank you very much LouD :) now works ok, spectacular!. I play tunes with usbsidberry, much better of deepsid website.
Without  DG467 chip, I can use a socket two?

**Reply by LouDnl** [reply]
> Thank you very much LouD :) now works ok, spectacular!. I play tunes with usbsidberry, much better of deepsid website. Without DG467 chip, I can use a socket two?

You can always use the second socket, even without the chip, except it will always be in stereo. SID 1 for left channel and SID 2 for right channel.
You should try VSID from Vice: https://github.com/LouDnl/Vice-USBSID/releases/tag/r45709

**Reply by matt199394** [reply]
Thanks again for the help and sharing this cool project!
Best regards
